LRQA-34098 | Master

acbbennett opened the pull request:
https://issues.liferay.com/browse/LRQA-34098

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

acbbennett commented:
Updated per comments on https://github.com/austinchiang/liferay-portal-ee/pull/139
Both PRs should utilize the same portlets. Poshi results and push comment attached to ticket also updated.

austinchiang commented:
No test changes needed.

austinchiang commented:
Pushed to BChan.
https://github.com/brianchandotcom/liferay-portal/pull/57058
